pkg/gateway/model_build_targetgroup.go

```diff
@@ -302,7 +313,11 @@ func (t *latticeServiceModelBuildTask) buildHTTPTargetGroupSpec(ctx context.Cont
 
 		} else {
 			glog.V(6).Infof("buildHTTPTargetGroupSpec, using service Import %v, err :%v\n", namespaceName, err)
-			return latticemodel.TargetGroupSpec{}, err
+			if !isDeleted {
+				//Return error for creation request only.
+				//For ServiceImport deletion request, we should go ahead to build TargetGroupSpec model
+				return latticemodel.TargetGroupSpec{}, err
```

Comment:
Return error for creation request only.
For ServiceImport deletion request, we should go ahead to build TargetGroupSpec model